### PR TITLE
Quantile count

### PR DESCRIPTION
**Pull request by poojke**

### CODE REVIEW

**Comment by poojke** [comment]
@AndreasKostler please let me know if you see any problems with these changes. Thx.

File: src/main/scala/au/com/cba/omnia/grimlock/scalding/Distribution.scala [hunk 1]
```diff
@@ -52,28 +55,36 @@ trait ApproximateDistribution[P <: Position with CompactablePosition] extends Fw
...
   }
 
-  type QuantileTuners[T] = TP2[T]
-  def quantile[S <: Position with ExpandablePosition, Q <: Position, W, T <: Tuner : QuantileTuners](slice: Slice[P],
+  type QuantileTuners[T] = T In OneOf[InMemory[NoParameters]]#
+    Or[InMemory[Reducers]]#
+    Or[Default[NoParameters]]#
+    Or[Default[Reducers]]
+  def quantile[S <: Position with ExpandablePosition, Q <: Position, T <: Tuner : QuantileTuners](slice: Slice[P],
     probs: List[Double], quantiser: Quantile.Quantiser, name: Locate.FromSelectedAndOutput[S, Double, Q],
```
Comment (AndreasKostler) [inline src/main/scala/au/com/cba/omnia/grimlock/scalding/Distribution.scala]: Just a comment - I really like this. It reads nicely :+1:

File: src/main/scala/au/com/cba/omnia/grimlock/scalding/Distribution.scala [hunk 2]
```diff
@@ -52,28 +55,36 @@ trait ApproximateDistribution[P <: Position with CompactablePosition] extends Fw
       .map { case (p, s) => Cell(p, Content(DiscreteSchema[Long](), s)) }
   }
 
-  type QuantileTuners[T] = TP2[T]
-  def quantile[S <: Position with ExpandablePosition, Q <: Position, W, T <: Tuner : QuantileTuners](slice: Slice[P],
+  type QuantileTuners[T] = T In OneOf[InMemory[NoParameters]]#
```
Comment (AndreasKostler) [inline src/main/scala/au/com/cba/omnia/grimlock/scalding/Distribution.scala]: Just a comment - I really like this. It reads nicely :+1:

**Comment by AndreasKostler** [comment]
Have you done any performance evaluation? I.e. does it degrade (much/at all)?
Otherwise gtg

**Comment by poojke** [comment]
@AndreasKostler Thanks for the review. Sorry now performance eval - quantiles performance tuning is blocked by shuffle speed investigations.

**Pull request merged**